llvm/test/CodeGen/RISCV/GlobalISel/legalizer/legalize-bswap-rv32.mir

```diff
@@ -2,7 +2,9 @@
 # RUN: llc -mtriple=riscv32 -run-pass=legalizer %s -o - \
 # RUN:   | FileCheck %s --check-prefix=RV32I
 # RUN: llc -mtriple=riscv32 -mattr=+zbb -run-pass=legalizer %s -o - \
-# RUN:   | FileCheck %s --check-prefix=RV32ZBB
+# RUN:   | FileCheck %s --check-prefix=RV32ZBB_OR_RV32ZBKB
```

Comment:
I think these can be simplified to CHECK?

Reply:
We are checking generated code in two scenarios: when zbb or zbkb is present and when it's not. So I think having these descriptions in the prefix is good.